### DISCUSSION TITLE
datePicker fast input with short year

### DISCUSSION THREAD

**Discussion by ritonglue** [primefaces/community#1149] [PrimeFaces]
Today the datePicker allows fast input with mask = true (and autocomplete=off showOnFocus=false showIcon=true)

My users would like to enter a date without writing the full year for example : 31/12/24 instead of 31/12/2024 (pattern is dd/MM/yyyy)
Of course the final display must be 31/12/2024.

Is it already possible ? If not I think this could be a great feature.

Thank you

**Comment by melloware** [comment]
have you tried making your pattern `dd/MM/yy` ?

**Comment by ritonglue** [comment]
Yes I tried but the final display is 31/12/24 which is not what I want.

As I said it is a "nice to have" feature to accelerate the input of dates.
